### PR TITLE
[WIP] IIN Documentation 

### PR DESCRIPTION
**Pull request by mwang87**

### CODE REVIEW

**Comment by lfnothias** [comment]
Can you merge that PR so I can start updating the doc ?

**Comment by lfnothias** [comment]
I would suggest to push these changes to the website already.

**Comment by robinschmid** [comment]
We should push it and polish it afterward.
What is the progress on this?

File: docs/fbmn-iin-msdial.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+**For GNPS**: Wang, M. et al. [Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking](https://doi.org/10.1038/nbt.3597). Nat. Biotechnol. 34, 828–837 (2016).
+
+**For MS-DIAL** 1): Tsugawa, H., Cajka, T., Kind, T., Ma, Y., Higgins, B., Ikeda, K., Kanazawa, M., VanderGheynst, J., Fiehn, O. & Arita, M. [MS-DIAL: data-independent MS/MS deconvolution for comprehensive metabolome analysis.](http://dx.doi.org/10.1038/nmeth.3393) Nature Methods 12, 523-526 (2015).
+
+**For MS-DIAL** 2): Lai, Z., Tsugawa, H., Wohlgemuth, G., Mehta, S., Mueller, M., Zheng, Y., Ogiwara, A., Meissen, J., Showalter, M., Takeuchi, K., Kind, T., Beal, P., Arita, M. & Fiehn, O. [Identifying metabolites by integrating metabolome databases with mass spectrometry cheminformatics.](http://dx.doi.org/10.1038/nmeth.4512) Nature Methods 15, 53-56 (2018).
+
+## Running the FBMN IIN workflow with MS-DIAL
+
+The integration of MS-DIAL and IIN is supported since version 2.99. After a standard processing data with MS-DIAL, the “Alignment results” can be exported for IIN analysis.
```
Comment (robinschmid) [inline docs/fbmn-iin-msdial.md]: version 4.1 (FBMN since version 2.99)

**Pull request merged**